shared/js/background/load.es6.js

```diff
@@ -1,20 +1,31 @@
 const browserWrapper = require('./$BROWSER-wrapper.es6')
 
-function JSONfromLocalFile (path, cb) {
-    loadExtensionFile({url: path, returnType: 'json'}, (res) => cb(JSON.parse(res)))
+let dev = false
+
+function JSONfromLocalFile (path) {
+    return loadExtensionFile({url: path, returnType: 'json'})
 }
 
-function JSONfromExternalFile (url, cb) {
-    try {
-        loadExtensionFile({url: url, returnType: 'json', source: 'external'}, (res, xhr) => cb(JSON.parse(res), xhr))
-    } catch (e) {
-        console.log(e)
-        return {}
-    }
+function JSONfromExternalFile (url) {
+    return loadExtensionFile({url: url, returnType: 'json', source: 'external'})
+}
+
+function url (url, cb) {
```

Comment:
`cb` is not called 🤔

Reply:
A previous iteration of this still used callbacks. Lemme remove.